src/main/java/uk/gov/digital/ho/pttg/api/AuditHistoryResource.java

```diff
@@ -54,6 +54,22 @@ public AuditHistoryResource(AuditHistoryService auditHistoryService, RequestData
         return result;
     }
 
+    @GetMapping(value = "/records", produces = APPLICATION_JSON_VALUE)
```

Comment:
"records" is rather ambiguous... why not also use "history"?  It's just another audit history search...

Reply:
Addressed